.happy/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "config_version": "v1",
+    "terraform_version": "0.13.5",
+    "default_env": "rdev",
+    "app": "hosted-cellxgene",
+    "default_compose_env": ".env.ecr",
+    "environments": {
+        "rdev": {
+            "aws_profile": "single-cell-dev",
+            "secret_arn": "happy/env-rdev-config",
+            "log_group_prefix": "/dp/rdev"
```

Comment:
Should this be something like `/hc/rdev` (hosted cellxgene) instead?

Reply:
yes, though I'm not sure this is even being used here, since it's infrastructure-related. Unless it's getting baked into the docker image.  Ideally, we can just delete this line if it's not used.